https/stats.js

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
I love everything about this file

Reply:
Ye, the overall stats are for frontend use and the other oens are for admin use.

For the frontend it should allow us to list how many folks have signed up, gifts sent, gifts recieved etc

Reply:
Agreed! Looking forward utilizing them